app/lib/steps/income_section.rb

```diff
@@ -5,27 +5,13 @@ def all_steps
         %i[employment housing_benefit housing_benefit_details benefits benefit_details other_income outgoings]
```

Comment:
I think we can remove `housing_benefit` `housing_benefit_details` and `outgoings` from these steps

Reply:
Good spot! Have removed.